str/str.go

```diff
@@ -104,16 +104,15 @@ func Between(subject string, from string, to string) string {
 }
 
 func Contains(haystack string, needle string) bool {
-	// TODO
-	return false
+	return strings.Index(haystack, needle) == -1
 }
 
-func ContainsInSlice(haystack string, needle []string) bool {
+func ContainsFromSlice(haystack string, needle []string) bool {
 	// TODO
 	return false
 }
 
-func ContainsAllInSlice(haystack string, needle []string) bool {
+func ContainsAllFromSlice(haystack string, needle []string) bool {
```

Comment:
What do you think of the following?
`ContainsBySlice()`
`ContainsAllBySlice()`

Reply:
Of course. I will change this. I had just worked for some minutes on it and simply pushed the changes. More to come when I have the time to do so.